myriad-scheduler/src/main/java/org/apache/myriad/scheduler/ServiceTaskFactoryImpl.java

```diff
@@ -232,7 +242,12 @@ public ExecutorInfo getExecutorInfoForSlave(FrameworkID frameworkId, Offer offer
         }
       }
     }
-    // this is actually an error condition - we did not have enough ports to use
+    //this is actually an error condition - we did not have enough ports to use
+    //TODO (hokiegeek2) does this need error handling then?
     return returnedPorts;
   }
-}
+  
+  private boolean isDefaultRole(Resource resource) {
+    return !resource.hasRole() || resource.getRole().equals("*");
+  }
```

Comment:
Agreed, I will make it so in another PR

On Wed, Jun 1, 2016 at 1:10 AM, Darin <notifications@github.com> wrote:

> In
> myriad-scheduler/src/main/java/org/apache/myriad/scheduler/ServiceTaskFactoryImpl.java
> <https://github.com/apache/incubator-myriad/pull/71#discussion_r65302676>:
>
> >      return returnedPorts;
> >    }
> > -}
> > +
> > +  private boolean isDefaultRole(Resource resource) {
> > +    return !resource.hasRole() || resource.getRole().equals("*");
> > +  }
>
> This is used elsewhere as well, worth adding to some static utility class?
> Maybe not this PR, but something to think about.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/71/files/37b4df2dffec60b2825ed21754a56673eb195b03#r65302676>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKSRgVry4VNuJTwfCM6veRmVSA4U41S5ks5qHRQrgaJpZM4Ib_Am>
> .
>